### PR TITLE
Moved instantiateCommand, callRuntimeAttributes, lookupFunction into JobDescriptor: closes #493, closes #483, closes #535

### LINKED ISSUE #483: Move lookupFunction to Backend
Likely entangled with #482

### LINKED ISSUE #493: Move instantiateCommand to Backend

### LINKED ISSUE #535: Move RuntimeAttributes from BackendCall into backend package
This has been partially done in #493/#536. But the backend.callRuntimeAttributes method needs completing.

### PR DESCRIPTION
**Pull request by cjllanwarne**

### CODE REVIEW

**Comment by cjllanwarne** [comment]
@kshakir @Horneth please!

File: engine/src/main/scala/cromwell/engine/backend/JobDescriptor.scala [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
 
+import cromwell.engine.backend.runtimeattributes.CromwellRuntimeAttributes
 import cromwell.engine.{CallEngineFunctions, WorkflowDescriptor}
 import cromwell.engine.workflow.{BackendCallKey, CallKey, FinalCallKey}
 import cromwell.webservice.WorkflowMetadataResponse
 import wdl4s._
 import wdl4s.values.WdlValue
 
+import scala.util.Try
+
```
Comment (Horneth) [inline engine/src/main/scala/cromwell/engine/backend/JobDescriptor.scala]: remove \n

File: engine/src/main/scala/cromwell/engine/backend/local/SharedFileSystem.scala [hunk 2]
```diff
@@ -229,6 +248,7 @@ trait SharedFileSystem { self: Backend =>
 
   /**
    * Try to localize a WdlValue if it is or contains a WdlFile.
+ *
```
Comment (kshakir) [inline engine/src/main/scala/cromwell/engine/backend/local/SharedFileSystem.scala]: Bad indentation

**Comment by kshakir** [comment]
Minor comments. LGTM :+1:

**Comment by Horneth** [comment]
:+1:

**Pull request merged**